src/SumoCoders/FrameworkUserBundle/Resources/translations/messages.nl.yml

```diff
@@ -17,6 +17,7 @@ user:
       admin: beheerder toevoegen
     settings: instellingen
     logout: uitloggen
+    switchBack: naar originele gebruiker
```

Comment:
don't we have english translations as well?

Reply:
Fixed 4df5cef